cluster/vsphere/util.sh

```diff
@@ -344,7 +344,7 @@ function kube-up {
 
   printf "Waiting for ${KUBE_MASTER} to become available..."
   until curl --insecure --user "${KUBE_USER}:${KUBE_PASSWORD}" --max-time 5 \
-          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta1/pods"; do
+          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta3/pods"; do
```

Comment:
Not necessarily in this PR, but we should define a shell constant for latest API version and use that in all our scripts like we use latest.Version in all our go code. Else we will have to do this again for moving from beta3 to v1
We can define that constant in https://github.com/GoogleCloudPlatform/kubernetes/blob/master/hack/lib/init.sh maybe.